lib/fromIntrospectionQuery.ts

```diff
@@ -14,13 +14,21 @@ export interface GraphQLJSONSchema6 extends JSONSchema6 {
 }
 
 export interface FromIntrospectionQueryOptions {
-  ignoreInternals?: boolean // true by default
+  ignoreInternals?: boolean // default: true
+  nullableArrayItems?: boolean // default: true
 }
+
 export const fromIntrospectionQuery = (
   introspection: IntrospectionQuery,
   opts?: FromIntrospectionQueryOptions
 ): JSONSchema6 => {
-  const options = opts || { ignoreInternals: true }
+  const options = {
+    // Defaults
+    ignoreInternals: true,
+    nullableArrayItems: true,
```

Comment:
@newhouse for back-compatibility reasons, I think we should have `nullableArrayItems` to `false` by default

wdyt?

Reply:
@wittydeveloper 

I think this comes down to a decision about versioning and feature exposure:
- For sure this new code path is a breaking change, so enabling by default AND not bumping to a new Major version is not OK.
- But it's also the "right" way things should be done, and we should steer people towards it, not away.

IMO, it may be worth doing 2 things right now:
1. Change the default here to `false`, as you suggest. then cut a new `minor` version and publish it.
2. Create a new branch / release candidate for a new major version but not yet publishing the new version. I think there are maybe a few other possibly-breaking things that we might want to tidy up before a major release. This way we can steer people towards the new, improved/corrected behavior.

What do you think about that? If you agree, we should maybe come up with a list of things that we might also want to get into a Major release change. I can think of maybe 2 or 3 things to at least consider.

We could also

Reply:
Agree with you, let's proceed this way! ⚡